pcsx2/Interpreter.cpp

```diff
@@ -524,7 +530,6 @@ static void intExecute()
 		{
 			// Avoid reloading every instruction.
 			u32 elf_entry_point = VMManager::Internal::GetCurrentELFEntryPoint();
-			u32 eeload_main = g_eeloadMain;
```

Comment:
Please don't remove this. I was going to address it in a PR on the weekend, but it blew out of scope and I didn't finish it.

The idea is for it to try to cache it in a register, rather than always loading from the global.